broadlink/switch.py

```diff
@@ -212,13 +212,31 @@ def check_nightlight(self) -> bool:
         payload = self.decrypt(response[0x38:])
         return bool(payload[0x4] == 2 or payload[0x4] == 3 or payload[0x4] == 0xFF)
 
-    def get_energy(self) -> int:
+    def get_energy(self) -> float:
+        """Return the energy state of the device."""
+        packet = bytearray(16)
+        packet[0] = 4
+        response = self.send_packet(0x6A, packet)
+        check_error(response[0x22:0x24])
+        payload = self.decrypt(response[0x38:])
+        return int.from_bytes(payload[0x4:0x7], "little") / 1000
```

Comment:
Maybe we should return the power in mW to have the same interface as SP4. This would make the implementation in Home Assistant much cleaner.

Reply:
I thought better and I think there is no reason for this breaking change. Better to perform all conversions here and return the result in W.